### ISSUE TITLE
DEFAULT_WIDTH undefined

### ISSUE THREAD

**Issue by binford2k** [pazdera/word_wrap#5]
When monkey patching string, the DEFAULT_WIDTH constant isn't set.

https://github.com/pazdera/word_wrap/blob/master/lib/word_wrap.rb#L17

```
2.0.0-p598 :001 > require 'word_wrap'
 => true
2.0.0-p598 :002 > 'foo bar'.wrap
NameError: uninitialized constant String::DEFAULT_WIDTH
	from /Users/ben/.rvm/gems/ruby-2.0.0-p598@mdl/gems/word_wrap-0.2.1/lib/word_wrap.rb:17:in `wrap'
	from (irb):2
	from /Users/ben/.rvm/rubies/ruby-2.0.0-p598/bin/irb:12:in `<main>'
```

**Comment by pazdera** [comment]
Hi Ben, Thanks a lot for reporting this.

I'll fix it as soon as I can and release an update.

**Comment by pazdera** [comment]
The problem should be fixed in [0.2.2](https://rubygems.org/gems/word_wrap/versions/0.2.2) which I've just released.

Thanks, Ben!
